optimas/explorations/base.py

```diff
@@ -615,3 +623,13 @@ def _save_exploration_parameters(self):
         file_path = os.path.join(main_dir, "exploration_parameters.json")
         with open(file_path, "w") as file:
             file.write(json.dumps(params))
+
+    def _set_manager(self, comms: str, libE_specs: Dict) -> bool:
```

Comment:
Could you add a docstring here with a short comment about why this is important? Something like "this is need when running with mpi comms to determine whether the this is the manager process".